src/schema/city/index.ts

```diff
@@ -115,6 +117,25 @@ const CityType = new GraphQLObjectType<any, ResolverContext>({
           artGuideUrl: {
             type: GraphQLString,
           },
+          shows: {
+            type: showConnection,
+            args: pageable({
+              sort: PartnerShowSorts,
+              status: EventStatus,
+            }),
+            resolve: async (
+              citySponsoredContent,
+              args,
+              { showsWithHeadersLoader }
+            ) => {
+              return loadData(args, showsWithHeadersLoader, {
+                id: citySponsoredContent.showIds,
+                include_local_discovery: true,
```

Comment:
do we need `has_location: true,` and `displayable: true` here or we assume those ids are good for being returned even if they are not displayable and don't have location?

Reply:
I think those flags are not necessary for this use case.

`has_location` isn't necessary because this query's purpose is to populate a list, not put pins on the map. (In fact you'll notice there's no `near` clause here either. That's because we specifically want to allow these to show up for their associated city, whether or not they are within the normal LD radius.)

`displayable: true` — some of these will be stub shows and will not have enough install shots to be officially `displayable`, so I don't think we need that, but I will double check and follow up. Thanks for raising.

Reply:
That makes sense, thanks Roop!